src/index.html

```diff
@@ -5,8 +5,402 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+
+      <nav  class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="tablets">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
```

Comment:
You can delete this section

Reply:
Yes, but I think there may be another section or block in the main part and I created a section. Should I delete the section or do I think so?